lib/tokens/key_fetch_token.js

```diff
@@ -8,6 +8,7 @@ module.exports = function (log, inherits, Token, P, error) {
     Token.call(this, keys, details)
     this.keyBundle = details.keyBundle
     this.emailVerified = !!details.emailVerified
+    this.tokenVerificationId = details.tokenVerificationId
```

Comment:
Is it still the case that we're not requiring verification of keyFetchTokens?  If so, we can probably remove this property here and elsewhere in the code.